docs/operator-manual/upgrading/2.4-2.5.md

```diff
@@ -49,6 +49,37 @@ following message:
 
 If you run `argocd app list` as admin, the list of Applications using deprecated plugins will be logged as a warning.
 
+### Remove the plugin name field
+
+Once you've set up a sidecar plugin that handles the same files as the legacy
+plugin (including the discover property so that argo-cd will know to select it),
+you should update each Application spec to remove the plugin name.
+
+E.g., if you had an application that looked like:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: guestbook
+spec:
+  source:
+    plugin:
+      name: pluginName  # Delete this line
+```
```

Comment:
Add something like <other fields> below name here and below plugin: below.

Reply:
? i don't understand

Reply:
```
    plugin:
      name: pluginName  # Delete this line
      <other fields>
```

```
    plugin:
      <other fields>
```